docs/computer/student_Reports/yash_pranshi_saloni_2020.md

```diff
@@ -0,0 +1,90 @@
+
+| Title   |      Author     |  Email-Id  |
+|:----------:|:-------------:|:------:|
+| Report for the tech stack learned till now at AUV club, ZHCET | Yash Agarwal | yashagarwal1214@gmail.com |
+| | Pranshi Jindal | apranshi11@gmail.com |
+| | Saloni Gupta | saloniguptasg12@gmail.com |
+
+# Git and Github
+
+**Objective**: To learn git and github.
+
+**Introduction**: Git is used to store the source code for a project and track the complete history of all changes to that code. It allows developers to collaborate on a project more effectively by providing tools for managing possibly conflicting changes from multiple developers. GitHub allows developers to change, adapt and improve software from its public or private repositories.
+
+**Equipments/Software used**: Gitbash
+
+**Procedure** : Followed some tutorials. [Reference](https://www.youtube.com/watch?v=WbwIoQYP6no)
+
+**Results** : Completed
+
+**Future work** : To collaborate with team on bigger projects.
+
+# Linux Command Interface
+
+**Objective**: To get comfortable with the Linux Command Line.
+
+**Introduction**: The Command Line Interface (CLI), is a non-graphical, text-based interface to the computer system, where the user types in a command and the computer then successfully executes it. The terminal is the platform or the IDE that provides the command line interface (CLI) environment to the user. The CLI terminal accepts the commands that the user types and passes to a shell. The shell then receives and interprets what the user has typed into the instructions that can be executed by the OS (Operating System)
+
+**Equipments/Software used**: Linux OS.
+
+**Procedure**: Followed some tutorials. [Reference](https://www.youtube.com/playlist?list=PLS1QulWo1RIb9WVQGJ_vh-RQusbZgO_As)
```

Comment:
While writing procedure... don't be ambigious by writing 'some'. Be straight forward